library/core/src/num/mod.rs

```diff
@@ -1410,13 +1420,23 @@ const fn from_str_radix_panic(radix: u32) {
 }
 
 macro_rules! from_str_radix {
-    ($($int_ty:ty)+) => {$(
+    ($signedness:ident $($int_ty:ty)+) => {$(
```

Comment:
```suggestion
    ($signedness:ident; $($int_ty:ty)+) => {$(
```
maybe? or some other separator to distinguish sign & types?

Reply:
It's quite clear in an IDE since they're colored different (ident vs type). So the two approaches seem about equivalent to me. @tgross35 any thoughts?

Reply:
^ to your question, `foo!(some_prefix u8 u16 u32 u64 u128; other_prefix i8 i16 i32 i64 i128)` (i.e. without commas) is a pretty common pattern for these implementation macros. Not that there is any specific reason, it doesn't matter much either way.